### ISSUE TITLE
Accessibility: Ambiguous view link on My Account orders table

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43628]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

On the My Account Orders page, the orders table contains a "View" button in the actions table that does not include any reference to the account number, either in visible text or an aria-label.
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/orders.php#L70

### Expected behavior

The View link should be unique so that when multiple orders are present in the table, it is possible to distinguish each View link from one another.

Add an aria-label that includes the order number to the link like this:
`<a href="https://shop.equalizedigital.com/my-account/view-order/518/" class="woocommerce-button wp-element-button button view" aria-label="View order number 518">View</a>`

### Actual behavior

Links are ambiguous

### Steps to reproduce

See code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
